### PR TITLE
Squarer-2

### PR DESCRIPTION
**Pull request by YazeedLoonat**
This creates the squarer game
relies on https://github.com/homegamesio/squish/pull/1 to function properly

### CODE REVIEW

File: src/games/squarer.js [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+    createNpc() {
+        this.movementQueue = [];
+        this.npc.forEach(npc => {
+            clearInterval(npc.interval);
+            this.base.removeChild(npc.id);
+        });
+        this.npc = [];
+        if (this.level > 9) {
+            console.log("good job");
```
Comment (prosif) [inline src/games/squarer.js]: is this the "win" state?

File: src/games/squarer.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                const temp = GameNode(color, null, {x: (10 * l) + 5, y: i}, this.defaultSize);
+                temp.interval = setInterval(() => {
+                    this.movementQueue.push({ npc: true, node: temp, direction: 'left' });
+                }, tickRate);
+                this.npc.push(temp);
+                this.base.addChild(temp);
+            }
+        }
+
+        for( let i = 55; i > 50 && i < 95; i+=5) {
```
Comment (prosif) [inline src/games/squarer.js]: Shouldn't `i` _always_ be above 50 in this loop?

File: src/games/squarer.js (hunk 1 above)
Comment (YazeedLoonat) [reply src/games/squarer.js:50]: it will be yes!

File: src/games/squarer.js (hunk 2 above)
Comment (YazeedLoonat) [reply src/games/squarer.js:67]: nice catch, updated, thank you!

**Pull request merged**